torch/distributed/_composable/fsdp/_fsdp_param_group.py

```diff
@@ -164,7 +168,10 @@ def _init_grad_divide_factors(self):
         if self._reduce_dtype == torch.float32:
             # Use NCCL's AVG op to divide after reduction since it is more
             # performant and fp32 has sufficient precision
-            self._grad_divide_factors: Optional[Tuple[float, float]] = None
+            self._grad_divide_factors: Union[Tuple[None, None], Tuple[float, float]] = (
+                None,
+                None,
```

Comment:
```suggestion
                None, None
```

Reply:
tried to write ``None, None`` but ``lintrunner -a`` put them into ``None, \n None again``

Reply:
No worries -- sorry for making you try